Pkg mgr settings

sbeaver-netgate opened the pull request:
Ready for review

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
pkg_mgr_settings Conversion complete

Ready for review

sbeaver-netgate committed:
Remove debug statement

SjonHortensius commented on usr/local/www/pkg_mgr_settings.php:
```diff
@@ -47,104 +47,90 @@
+<script>
 //<![CDATA[
+events.push(function() {
+	// Disables the specified input element
```
Why not use `>toggles()` for this instead?

sbeaver-netgate committed:
Use toggles as suggested

Thanks,

The toggle system was a WIP when I did this page I think. It works
nicely now in this instance.

sbeaver-netgate committed:
Remove unneeded gettext
